src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/DeleteManagedOrganizationUserAccountCommand.cs

```diff
@@ -89,21 +105,19 @@ public async Task DeleteUserAsync(Guid organizationId, Guid organizationUserId,
 
                 await CancelPremiumAsync(user);
 
-                userDeletionResults.Add((orgUserId, orgUser, user, string.Empty));
+                userDeletionResults.Add((orgUserId, orgUser, user, null));
             }
-            catch (Exception ex)
+            catch (Exception exception)
             {
-                userDeletionResults.Add((orgUserId, orgUser, user, ex.Message));
+                userDeletionResults.Add((orgUserId, orgUser, user, exception));
```

Comment:
I understand why you made this change, but given my feedback above I think it's unnecessary. This change also means the try/catch pattern is leaking out to other code (because we need it to give us the `Exception`) which I'd prefer not to do.